src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplatePhrase/PhrasalTemplatePhrase.tsx

```diff
@@ -31,26 +31,38 @@ import {
 import { PhrasalTemplatePhraseProps } from './PhrasalTemplatePhrase.types';
 import { getFieldPlaceholders, getNewDefaultField } from '../PhrasalTemplate.utils';
 import { KEYWORDS } from '../PhrasalTemplateField/PhrasalTemplateField.const';
+import { PhrasalTemplateResponseValuePhraseSchema } from '../../../../../../pages/BuilderApplet/BuilderApplet.schema';
 
 export const PhrasalTemplatePhrase = ({
   name = '',
   responseOptions = [],
   index = 0,
   onRemovePhrase,
+  field,
 }: PhrasalTemplatePhraseProps) => {
   const { t } = useTranslation('app');
+  // Use `useEffect` to trigger validation when `fields` or form state changes
   const [removePopupOpen, setRemovePopupOpen] = useState(false);
   const [previewPopupOpen, setPreviewPopupOpen] = useState(false);
-  const [addItemMenuAnchorEl, setAddItemMenuAnchorEl] = useState<null | HTMLElement>(null);
+  const [previewPhraseDisabled, setPreviewPhraseDisabled] = useState(false);
 
+  const [addItemMenuAnchorEl, setAddItemMenuAnchorEl] = useState<null | HTMLElement>(null);
   const phraseFieldsName = `${name}.fields`;
-  const { control, setValue, getValues, formState } = useCustomFormContext();
-
+  const { control, setValue, getValues, formState, watch } = useCustomFormContext();
```

Comment:
Nit: There's an ESLint warning about an unused variable, maybe when debugging your code earlier? Try to notice when you introduce ESLint warnings. (Tip: Whenever I submit a PR, I usually review my PR's **Files changed** tab in GitHub to do a sanity check and catch debugging code that I meant to delete, because it happens a lot!)

It's true there are many existing ESLint warnings still in the codebase, but we are actively working to eliminate them as we encounter them. At the minimum, avoid introducing new ones. 😇🙏